showcase/src/app/shared/effects/stark-error-handling.effects.ts

```diff
@@ -11,7 +11,7 @@ import uniqueId from "lodash-es/uniqueId";
  */
 @Injectable()
 export class StarkErrorHandlingEffects {
-	private _starkToastNotificationService: StarkToastNotificationService;
+	private _starkToastNotificationService?: StarkToastNotificationService;
```

Comment:
But we do the `if (typeof this._starkToastNotificationService === "undefined")`, so that does mean we expect it to not be defined in some cases right? 🤔

Reply:
Indeed, but that is just the first time when the service has not been injected... then the service will always be defined.

But ok... either `?` or `!` are fine for me... this doesn't affect the logic in any case ;)